storage/innobase/srv/srv0start.cc

```diff
@@ -1507,8 +1507,6 @@ dberr_t srv_start(bool create_new_db)
 			: recv_recovery_from_checkpoint_start(flushed_lsn);
 		recv_sys.close_files();
 
-		recv_sys.dblwr.pages.clear();
```

Comment:
`recv_sys.dblwr.pages` would seem to waste memory until `recv_sys.close()` is invoked on server shutdown. The collection serves no useful purpose after recovery has finished. Can we clear this near the end of `recv_sys_t::apply()` inside the `if (last_batch)` section while we are holding `recv_sys.mutex`?